package.json

```diff
@@ -55,7 +57,10 @@
     "@eslint/eslintrc": "^2.0.2",
     "@eslint/js": "^8.37.0",
     "@release-it/conventional-changelog": "^4.3.0",
-    "@typescript-eslint/parser": "^5.36.2",
+    "@types/eslint": "^8.56.2",
+    "@types/estree": "^1.0.5",
+    "@typescript-eslint/parser": "^6.20.0",
```

Comment:
Is there a strategy I should take around these `devDependencies` not being recognized by `node/no-missing-require`? 

```plaintext
$ npm run lint:js

> eslint-plugin-eslint-plugin@5.2.1 lint:js
> eslint --cache --ignore-pattern "**/*.md" .


/Users/josh/repos/eslint-plugin-eslint-plugin/lib/rules/no-property-in-node.js
  3:33  error  "@typescript-eslint/utils" is not found  node/no-missing-require

/Users/josh/repos/eslint-plugin-eslint-plugin/tests/lib/rules/no-property-in-node.js
  3:32  error  "@typescript-eslint/rule-tester" is not found  node/no-missing-require

/Users/josh/repos/eslint-plugin-eslint-plugin/tests/lib/rules/require-meta-docs-description.js
  298:27  error  "@typescript-eslint/parser" is not found  node/no-missing-require

/Users/josh/repos/eslint-plugin-eslint-plugin/tests/lib/utils.js
  10:40  error  "@typescript-eslint/parser" is not found  node/no-missing-require

✖ 4 problems (4 errors, 0 warnings)
```

Left to my own devices I'd try adding an override for test files that sets `settings.allowedModules`. And maybe file an issue on `eslint-plugin-n` to add a rule option for pulling in `devDependency`/`peerDependencies`/etc.

Reply:
Not immediately sure but a separate PR for `eslint-plugin-n` is welcome.

Reply:
Ah, removing the dependencies resolved the issue. Yay!